docker/odklite/Dockerfile

```diff
@@ -76,6 +77,14 @@ COPY scripts/check-rdfxml.sh /tools/check-rdfxml
 # Force Git to accept working on a repository owned by someone else
 RUN echo "[safe]\n    directory = /work" >> /etc/gitconfig
 
+# Make sure we run under an existing user account with UID/GID
+# matching the UID/GID of the calling user
+COPY scripts/entrypoint.sh /usr/local/sbin/odkuser-entrypoint.sh
+RUN chmod 755 /usr/local/sbin/odkuser-entrypoint.sh
+ENTRYPOINT ["/usr/local/sbin/odkuser-entrypoint.sh"]
```

Comment:
Will this have any consequences to GitHub actions using ODK and people that use the docker container without the run.sh script?

Reply:
Same consequences as for all other users: the processes that run within the container will run under the identify of an unprivileged user.

If they need those processes to run with root privileges and can’t use the `run.sh` script for any reason, then it’s up to them to explicitly pass the `ODK_USER_ID=0` environment variable when they invoke Docker.